From 0fae2b0206adef1305d49ac69f9229bd617383dc Mon Sep 17 00:00:00 2001
From: Noam Camus <ncamus@habana.ai>
Date: Mon, 11 Nov 2019 18:53:33 +0200
Subject: [PATCH] [SW-6379] goya2: remove call to printf() from main()

This helps to reduce binary image from 59KB to 31KB

Change-Id: I6f4a357efd9dbfb04c16fc943ba25d57a8767df4
---

diff --git a/arcboot.c b/arcboot.c
index 896dcf9..91ade8b 100644
--- a/arcboot.c
+++ b/arcboot.c
@@ -37,7 +37,7 @@
 	err = elp_secureboot_phase0b(sb_proxy_read, test_phase0b_data, test_phase0b_data_len, test_public_key, test_public_key_len,
 			test_phase0b_sig, test_phase0b_sig_len, &st, block_buf, sizeof block_buf);
 
-	printf("SB Phase0b Test: %s\n", (err || !st) ? "FAILED" : "PASSED");
+	//printf("SB Phase0b Test: %s\n", (err || !st) ? "FAILED" : "PASSED");
 	if (err || !st) {
 		return -1;
 	}
diff --git a/arcboot_rom.c b/arcboot_rom.c
index 896dcf9..91ade8b 100644
--- a/arcboot_rom.c
+++ b/arcboot_rom.c
@@ -37,7 +37,7 @@
 	err = elp_secureboot_phase0b(sb_proxy_read, test_phase0b_data, test_phase0b_data_len, test_public_key, test_public_key_len,
 			test_phase0b_sig, test_phase0b_sig_len, &st, block_buf, sizeof block_buf);
 
-	printf("SB Phase0b Test: %s\n", (err || !st) ? "FAILED" : "PASSED");
+	//printf("SB Phase0b Test: %s\n", (err || !st) ? "FAILED" : "PASSED");
 	if (err || !st) {
 		return -1;
 	}
